docs/sections/user_guide/yaml/tags.rst

```diff
@@ -23,6 +23,24 @@ Or explicit:
 
 Additionally, UW defines the following tags to support use cases not covered by standard tags:
 
+``!datetime``
+^^^^^^^^^^^^^
+
+Converts the tagged node to a Python ``datetime`` object. For example, given ``input.yaml``:
+
+.. code-block:: yaml
+
+   date1: 2024-09-01
+   date2: !datetime "{{ date1 }}"
+
+.. code-block:: text
+
+   % uw config realize -i ../input.yaml --output-format yaml
+   date1: 2024-09-01
+   date2: 2024-09-01 00:00:00
+
+The value provided to the tag must be in ISO 8601 format to be interpreted correctly by the ``!datetime`` tag.
```

Comment:
You might have decided against it for a good reason, but one more gentle plea for a link to https://docs.python.org/3/library/datetime.html#datetime.datetime.fromisoformat here, maybe like [ISO 8601 format](https://docs.python.org/3/library/datetime.html#datetime.datetime.fromisoformat), for examples of that and, especially, the notes in the Python docs about where its support differs from the standard.

Reply:
It's a great suggestion and I just let it slip through the cracks. I'll make the mod before I merge,  but that won't be until this afternoon.